.github/workflows/move-into-oss.yml

```
@@ -0,0 +1,57 @@
+name: Displaying jekyll in Docusaurus
+
+on:
+  push:
+    branches:
+      - github-actions
+
+jobs:
+  changed_files:
+    runs-on: ubuntu-latest  # windows-latest || macos-latest
```

```suggestion
    runs-on: ubuntu-latest
```
you can remove the others